cmd/insprd/memory/brokers/brokers.go

```diff
@@ -76,7 +87,9 @@ func (bmm *BrokerMemoryManager) SetDefault(broker string) error {
 		return ierrors.NewError().Message("broker %s is not configured on memory", broker).Build()
 	}
 
-	mem.Default = (broker)
+	logger.Info("SETTING BROKER TO DEFAULT BROKER", zap.String("BROKER", broker))
+
```

Comment:
might be ok to delete the first use of the `logger`

Reply:
Oops heh